Evolutionary theory predicts that related organisms will share similarities that are derived from common ancestors 
Similar characteristics due to relatedness are known as homologies 
Homologies can be revealed by:
	Comparing the anatomies of different living things 
	Looking at cellular similarities and differences 
	Studying embryological development
	Studying vestigial structures within individual organisms
Structural morphology
	Comparative anatomy
		Individual organisms contain, within their bodies, abundant evidence of their history
		The existence of these features is explained by evoloution
		Organisms that are closely related to one another share many anatomical similarities
	Homologous structures
		Similar structures are called homologous structures that can be traced back to a similar set of genes inherited over time. They have a fundamental similarity but may have different functions
		This is often evidence for divergent evolution where the one common ancestral group splits into distinct species with their own variations of a feature
	Transitional fossils
		Transition fossils in the fossil record can show intermediates of homologous structures, demonstrating the evolution of structural intermediates 
		Homologous structures do not change dramatically overnight and transitional fossils demonstrate the slow evolution of traits 
	Analogous structures
		Analogous structures share a function but cannot be tracked to the same or similar genes for their similarity
		For example wings serve the same function but have vastly different structures between different species
		This illustrates convergent evolution as unrelated species have become structurally more alike due to exposure to shared selection pressures
	Vestigial structures
		Structures that may have served an evolutionary purpose in the past but are no longer used
		An example are wings on a flightless bird. 
	Comparative embryology
		Another difficulty in comparing traits between species rests on the fact that homologous structures not present in the adult organism often do appear in some stage of embryonic development
		Species that bear little resemblance in their adult form may have strikingly similar embryonic stages
		For example, in humans, the embryo passes through a stage in which it has gill structures like those of the fish from which all terrestrial animals evolved 
		For a large portion of its development the human embryo also possesses a tail, much like those of our close primate relatives. This tail is usually reabsorbed before birth, but occasionally children are born with the ancestral structure intact
Molecular Homology
	Evolutionary relationships
		Structural morphology indicates how physical adaptations have changed over time
		DNA and amino acid sequences are used to work out level of relatedness
			Molecular evidence for evolution can indicate relationships between many species that might not have a similar structural morphology
		DNA and protein sequences evolve at a rate that is relatively constant over time and among different organisms 
			Referred to as a Molecular clock
				Molecular clocks can be used to work out how long ago two species diverged as well as the relatedness of different species 
	DNA sequences
		Molecular evidence involves identifying conservation in DNA and protein sequences as a basis for determining evolutionary relationships
		DNA could be nuclear DNA or mitochondrial DNA (mtDNA)
			Nuclear DNA is found in the nucleus and inherited from both parents
			mtDNA found in mitochondria that is inherited from mothers
		![](https://lh7-us.googleusercontent.com/slidesz/AGV_vUfGpgS1Grtj8DDQqxHZwKF5yEckG9Z1SqUHgSutR8XP9UXTSEE8bQjDHtzqSOPRsbSAKLCQsHUJqKZxJRWIruyxBurReXNB9X3_AId8nl9iQ5ZFT4Dq-47i3Dt_Y7Yb38Awgpiljz5i5zpBV3LnYhPYTTOlEi2yGPp0imiRutjUnKkBN28iNA=s2048?key=4TDG6sTSoIsn6F2h13QubQ)
		Gene conservation
			With the sequencing of a variety of organismal genomes, it has been found that many organisms that are not closely related, still retain particular genes in common
			These genes are said to be conserved and typically serve a critical role in controlling organismal development
			Examples of such genes that are ultra-conserved include: 
				Regulatory genes (e.g. coding for protein-based hormones)
				Genes encoding for tRNA (needed for translation to occur)
				Genes involved in ATP production (e.g. coding for the cytochrome proteins in the electron transport chain)
			The number of conserved genes can be used to show evolutionary relationships between organisms
			Conserved genes
				Nuclear or mitochondrial DNA of the same trait can be compared (e.g. eye colour)
				The more similar the DNA the closer related the organism
				Non-coding DNA provides the best means of comparison as mutations will occur more readily in these sequences. 
			mtDNA
				An important tool for tracing evolutionary relationships within a species
				Mitochondria DNA is structurally identical to nuclear DNA, but differs in a number of important ways
					Descent via maternal line
						mtDNA is inherited from the mother only, providing a much more direct genetic lineage
					Lack of recombination
						As mtDNA is passed directly from the mother, no recombination occurs
					Higher mutation rate
						Mitochondria produce reactive oxygen species which cause non-conserved sequences to mutate at a faster rate
					High copy number
						As every cell has multiple mitochondria, larger amounts of mtDNA can be gathered for sampling
				Because of these differences, mtDNA sequencing is ideal for comparing organisms within a species or those who have diverged in a relatively short time (~20 million years)
				Distinctive human mtDNA sequences have been found in different global populations and have been used to divide the human population into distinct haplogroups
				A haplogroup is a combination of alleles at different chromosomal regions that are closely linked and that tend to be inherited together.
				All humans are thought to be able to trace their origins back to one of seven super-haplogroups derived from an original haplogroup about 200,000 years old â€“ the 'mitochondrial Eve'
				![](https://lh7-us.googleusercontent.com/slidesz/AGV_vUe2klVa0pnuQSrFOm2Qkm2pXTnObLPiiQ0ErNPvPiTcFb7cq_UWoyEiMO6zHH0l5Re8CFUWQNkpttD1pX22yomuC6iK4Em8_CLYVnYDLwM0DH2b2IHaGgG4r6fCQzL3VkDaF41WiRLyA64hOALAM69cjjbsqxJoINHigDVELtQaH8OI9sHEVdY=s2048?key=4TDG6sTSoIsn6F2h13QubQ)
	Amino acid sequences
		Substitution mutations are common in DNA sequences.			
		Protein analysis allows us to compare conserved amino acid sequences.			
		This is especially useful for comparing proteins between species that are functionally the same but may have some small differences.
			![AGV_vUeyN7tSuKTrjogc1GeIB4Jel6OWr4z9kZSFuItxQjNmpHx0og3iQQz7-fZb0_BeFeCxtwCDW9WMWAjWx0lUJMfjWpXbZnlV9xfcU_7HSn0M4BgwfY51pbvzd8m8C0WFXtomnlnz-10MTVNQczxPxIRp1ZV8jGNXiBx1lKq41FZuPd9imXYCfdw=s2048?key=4TDG6sTSoIsn6F2h13QubQ|390](https://lh7-us.googleusercontent.com/slidesz/AGV_vUeyN7tSuKTrjogc1GeIB4Jel6OWr4z9kZSFuItxQjNmpHx0og3iQQz7-fZb0_BeFeCxtwCDW9WMWAjWx0lUJMfjWpXbZnlV9xfcU_7HSn0M4BgwfY51pbvzd8m8C0WFXtomnlnz-10MTVNQczxPxIRp1ZV8jGNXiBx1lKq41FZuPd9imXYCfdw=s2048?key=4TDG6sTSoIsn6F2h13QubQ)
		Protein analysis
			Cytochrome C is a molecule that has been used to study evolution. It is vital to the respiratory pathway. All organism break down glucose to respire, so this enzyme will be present
			Can take multiple forms without drastically altering its functions
				Lots of neutral mutations at a steady rate makes it a good molecular clock 
			Common sequences in cytochrome c are seen in more related organisms
		Molecular clocks
			Some genes or protein sequences may accumulate mutations at a relatively constant rate (e.g. 1 change per million years)
			If this rate of change is reliable, scientists can calculate the time of divergence according to the number of differences
				E.g.  If a gene which mutates at a rate of 1 bp per 100,000 years has 6 bp different, divergence occurred 600,000 years ago
			This concept is called the molecular clock and is limited by a number of factors:
				Different genes or proteins may change at different rates (e.g. haemoglobin mutates more rapidly than cytochrome c)
				The rate of change for a particular gene may differ between different groups of organisms
				Over long periods, earlier changes may be reversed by later changes, potentially confounding the accuracy of predictions
		Phylogenetic Trees
			Reveals the connection between organisms with their ancestors and descendants.
			A phylogenetic tree is a branching tree-like diagram which explains the phylogenetic relationships between organisms with the amount of evolutionary distance. 
			There are two main types of phylogenetic trees known as rooted and unrooted. 
				A rooted tree shows the oldest common ancestor of the tree. Has nodes to indicate common ancestors
				.![https://www.ncbi.nlm.nih.gov/Class/NAWBIS/Modules/Phylogenetics/images/phylo1003.gif](https://lh7-us.googleusercontent.com/slidesz/AGV_vUdk0CZWjMIBRQ0nFYuYv93dkdmUkAx6NGFfElSQqYVBOuUDXUIfw2KwXgSza9X8NglE34cigNA8kptD0YuMHadZBdEvZNBCsH6DzXnksi55cgoxzi7npzC6XDs2RBHP6I5hB6q30MZRCKQm2yxAy8zspVLWV1dV1fDtTHKDMbhgJ5vdqULT7Cw=s2048?key=4TDG6sTSoIsn6F2h13QubQ)
				An unrooted phylogenetic tree does not show an ancestral root. Distances between species show degrees of relatedness.
				![https://www.ncbi.nlm.nih.gov/Class/NAWBIS/Modules/Phylogenetics/images/phylo1002.gif](https://lh7-us.googleusercontent.com/slidesz/AGV_vUeOb5hjuVGDy7_1S3SqDArZZ6f-BbVpsB5XpQNOG7xFtFdZnZfwx-a9mTl1Lzahlsw8cSFPMfY2OXRHRMqK0VzGlomr8apj70Q9szRMpp2LjfCGLBzi8BcJi0STljzwNARyNCd-HEjmXzuF-4En7X-rPSxEO6FUcgfjrgvIFfIwbIi3wjUiuoY=s2048?key=4TDG6sTSoIsn6F2h13QubQ)
			The rooted phylogenetic tree
				A phylogenetic tree, or evolutionary tree, represents the evolutionary relationships among a set of organisms or groups of organisms.
				The tips of the tree represent groups of descendent taxa (often species) and the nodes on the tree represent the common ancestors of those descendants
				Shows the increasing complexity of new species
				Extinct species stop showing descendants on the phylogenetic tree
				![http://www.bio.miami.edu/dana/pix/evolutionary_tree.jpg](https://lh7-us.googleusercontent.com/slidesz/AGV_vUfGl-5XG-85VLBZuRPj1j4tgy36Xp6ub38B1wcR_9j8HXogSA4gxNHiIH_gUiurQyhwkwMCcVaNKapn0lJuHHchLEOsfLPaFwcTg6S5TUl3G6BbKLcxX_5xR1hv8NpOxrkvKhgxHVEe6ZrngIO_Q25KjoVdaLRv-0umTIzGVd3GqcWcIcoVE6k=s2048?key=4TDG6sTSoIsn6F2h13QubQ)
			The unrooted phylogenetic tree
				An unrooted phylogenetic tree is a phylogenetic diagram which lacks a common ancestor or a basal node. 
				This type of a tree does not indicate the origin of evolution of the groups of interest. 
				It depicts only the relationship between organisms irrespective of the direction of the evolutionary time line. 
				Therefore, it is difficult to study the evolutionary relationships of the groups with respect to time using an unrooted tree.
				![](https://lh7-us.googleusercontent.com/slidesz/AGV_vUfVSSXQaLWTxuoPcCeZfmLnpR4jxD8py6qPe0utveaXlcbKqQNsbhdrnkbaUWLIlnEOFgNTMEstn9ylKCyMxXRdpn3xp_jWJXDJADW08B4SZWf8dd86e1oR-CXL_WztVHXsZzOTP9q9JgWHAcOLSVk6WY6J5wt46GVPyHssDgcLwMCKRaCGsQ=s2048?key=4TDG6sTSoIsn6F2h13QubQ)
			![DiagramDescription automatically generated with medium confidence](https://lh7-us.googleusercontent.com/slidesz/AGV_vUdF8fMz3vDqdYcQ_qYR0nHVVPENvi8EWvJMGBsXnxtFz_g4R5R-N1JLu1QM4IFbWhLIUHT0h_qDnxsPk5OQAZVZMoKp0RFUhV5AnV_ockCsf76sGumrzHolmMxyhm53cfR8lj4LBQzkKd5uwjL91j2tTWBgXjL-BZAfGwLeP0bukR9u3Zi6YNQ=s2048?key=4TDG6sTSoIsn6F2h13QubQ)
			![](https://lh7-us.googleusercontent.com/slidesz/AGV_vUflTbF7l-7-YuVloqVhOnXvVEMzyLH7tUxStMYfrqPB3UZJX3mnQ9V8XVDpt65wlpFGm-8xV9Ezbp2hO79XwO_fOZ5ksVAPJaW459dX6_PP8tPkeh7dHevQFtdohJ9ry9LspJqOXpIcaUlTVRTT_Nt4-jUUMfCk1IjtR8nnV4zKVEwlx8Fd2Q=s2048?key=4TDG6sTSoIsn6F2h13QubQ)
			Cladogram
				The main difference between cladogram and phylogenetic tree is that cladogram is an evolutionary tree with branches with equal distance, showing the relationship between a group of clades whereas phylogenetic tree is an evolutionary tree showing an estimate of phylogeny where the distance of the each branch is proportional to the amount of inferred evolutionary change. 
			Determining degree of relatedness
				Nodes indicate a speciation event
				The number of nodes donâ€™t really matter. What matters more is when the most recent common ancestor between species exists
				![[evidence of relatedness-20240722090536177.webp]]
			Constructing phylogenetic trees
				1. Identify homologous molecular sequences to compare (often the hardest step for distantly related species). We need to consider suitable genes or amino acid sequences that are reliable, found amongst the species weâ€™re investigating and 
				2. Copy the molecular sequence (DNA or amino acid) for each of the selected organisms. These will go into computer databases dedicated to this field of work (bioinformatics)
				3. Align these sequences. The computer program will line up commonalities between the sequences to determine the degree of relatedness. This data is then used by the computer program to construct a diagram
			Summary
				Phylogenetic trees give diagrammatic representation of relatedness between species
				The data used to construct a phylogenetic tree can change what the tree looks like
				Trees can be rooted or unrooted. Rooted is more common and shows proportional degrees of relatedness and likely times when groups speciated from common ancestors
				The longer a group has separated from another, the more time there is for genetic differences to accumulate











